content/en/blog/_posts/2021-12-16-kubernetes-is-moving-on-from-dockershim.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: blog
+title: "Kubernetes is moving on from Dockershim: commitments and next steps"
+date: 2021-12-16
+slug: kubernetes-is-moving-on-from-dockershim
+---
+
+**Authors:** Sergey Kanzhelev (Google), Jim Angel (Google), Davanum Srinivas (VMware), Shannon Kularathna (Google), Chris Short (AWS), Dawn Chen (Google)
+
+Kubernetes is removing dockershim in upcoming version 1.24. Please welcome new
+container runtimes with more exciting features, a smaller kubelet, and faster
+engineering velocity for new features. If you [use Docker as a Container Runtime](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use/)
+for Kubernetes, get ready to migrate from dockershim! To check if you're
+affected, refer to [Check whether dockershim deprecation affects you](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/).
+
+## Why we’re moving away from dockershim
+
+Docker was the first container runtime used by Kubernetes. This is one of the
+reasons why Docker is so familiar to many Kubernetes users and enthusiasts.
+Kubernetes uses dockershim as a container runtime interface (CRI) to let system
+components (like the kubelet) talk to Docker. It was built into Kubernetes code.
+Removing dockershim inevitably affects the entire CNCF family of projects beyond
+Kubernetes. Dependencies on Docker and dockershim have crept into various tools
+and projects in the Kubernetes ecosystem.
+
+There are a few container runtimes today that are often lighter, faster and more
```

Comment:
Nit. Oxford comma.

There are a few container runtimes today that are often lighter, faster, and more secure than when Docker was the only choice